Tests/test_file_eps.py

```diff
@@ -404,3 +404,16 @@ def test_timeout(test_file):
         with pytest.raises(Image.UnidentifiedImageError):
             with Image.open(f):
                 pass
+
+
+@pytest.mark.parametrize(
+    ("trailer_image_file", "header_image_file"),
+    [("Tests/images/zero_bb_trailer.eps", FILE1)],
+)
+def test_boundary_box_in_trailer(trailer_image_file, header_image_file):
+    # Check whether boundary boxes which are specified in the trailer are parsed
+    # just like boundary boxes specified in the header
+    with Image.open(trailer_image_file) as trailer_image, Image.open(
+        header_image_file
```

Comment:
```suggestion
def test_boundary_box_in_trailer():
    # Check whether boundary boxes are parsed in the same way
    # when specified in the header or the trailer
    with Image.open("Tests/images/zero_bb_trailer.eps") as trailer_image, Image.open(
        FILE1
```
If there's only one parameter, it doesn't seem necessary to parametrize the test.